### PR TITLE
Initial implementation for reading Long Term Statistics (LTS) res1d files.

### PR DESCRIPTION
**Pull request by gedaskir**

### CODE REVIEW

**Comment by ecomodeller** [comment]
@gedaskir I am happy to review the PR once the test passes.

**Comment by gedaskir** [comment]
As of now we assume that for python version <= 3.8 we support pythonnet 2.5.2. The failure was related to pythonnet 2.5.2 handling enums as simple integers. I wonder if you should ditch pythonnet 2.5.2 support and migrate to pythonnet version >= 3.0.

**Comment by gedaskir** [comment]
@ecomodeller I changed the LTS result files for smaller ones and force pushed the change  :-).

File: mikeio1d/res1d.py [hunk 1]
```diff
@@ -136,6 +138,11 @@ def _add_node(self, node_id):
...
         self._data_subfilter.Catchments.Add(catchment_id)
 
+    def _is_lts_result_file(self):
+        # For pythonnet version > 3.0 it is possible to call
+        # return self._data.ResultType.Equals(ResultTypes.LTSEvents)
+        return int(self._data.ResultType) == int(ResultTypes.LTSEvents)
+
     #endregion File loading
 
     def read(self, queries=None):
```
Comment (ecomodeller) [inline mikeio1d/res1d.py:148]: Not directly related to this PR, but while you have the box open. I would add a couple of examples to the read docstring. To read all data is obvious, but not how to use a query.

So one example of a query would be nice to have.

**Review by ecomodeller — APPROVED** [review]

File: mikeio1d/res1d.py (hunk 1 above)
Comment (gedaskir) [reply mikeio1d/res1d.py:148]: It is a good suggestion. Also the Res1D class would benefit a docstring with all it's input parameters. I will add it.

Comment (gedaskir) [reply mikeio1d/res1d.py:189]: @ecomodeller Thanks for the review! I will merge the changes in now. I will use the principle that one review is enough if it is simple changes like this.

**Pull request merged**